### ISSUE TITLE
Where/how to indicate src directory?

### ISSUE THREAD

**Issue by tylerecouture** [GameDrivenDesign/docker-godot-export#8]
Giving this a stab, very cool!

But, my godoto source is in a directory, no at the github repo's root, adn I'm not familiar enough with docker volumes to fix it:

```
Setting environment variables from .travis.yml
$ export GAME_NAME="Panjwayi"
rvm
0.58s$ rvm use default
ruby.versions
$ ruby --version
install
23.64s$ docker pull gamedrivendesign/godot-export
$ docker run -e EXPORT_NAME="HTML5"           -e OUTPUT_FILENAME="index.html"           -v $(pwd)/panjwayi:/build/src -v $(pwd)/output/html5:/build/output   gamedrivendesign/godot-export
ERROR: _load_data: Condition "!f" is true. Returned: ERR_CANT_OPEN
   At: scene/resources/texture.cpp:502.
ERROR: _load: Condition "found" is true. Returned: RES()
   At: core/io/resource_loader.cpp:278.
```

Here's my repo, with all the godot source in the panjwai directory:
https://github.com/Weingardt-Games/panjwayi

**Comment by tom95** [comment]
Just adapting the -v already sounds like a good idea to me without thinking about it too much... Further, the error would have likely been a bit more explicit had it not found anything. So, I might be wrong, but my guess is that one of your resources is referring to a broken link for example. If you run docker locally from the command line, you should be able to see the same error I would assume, should it actually be a problem in your repository. Could you give that a quick try?

**Comment by tylerecouture** [comment]
When I try it locally, it creates an empty  `/output/html5` directory and then hangs?

```
$ export GAME_NAME="Panjwayi"
$ docker run -e EXPORT_NAME="HTML5" -e OUTPUT_FILENAME="index.html" -v $(pwd)/panjwayi:/build/src -v $(pwd)/output/html5:/build/output   gamedrivendesign/godot-export
```

**Comment by tylerecouture** [comment]
Note that this works for me locally (i..e the game at least builds and exports properly as is)

~/apps/Godot_v3.2.1-stable_x11.64 --export HTML5 --path ~/Godot/panjwayi/panjwayi ~/Godot/panjwayi/output/html5/panjwayi

**Comment by tom95** [comment]
Any possibility that you haven't pushed something that is important? Fastest way to check would be to do a fresh git clone and try there once.

**Comment by tylerecouture** [comment]
Only thing different is the ouput directory it created:
```
$ git pull
Already up to date.
$ git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	output/

nothing added to commit but untracked files present (use "git add" to track)
tylere@sparagus:~/Godot/panjwayi$ 
```

**Comment by tom95** [comment]
Hey, sorry for the late reply! I see you switched to GH actions in your repo, so you no longer have need for this repo? Or does it do not quite the same?

**Comment by tylerecouture** [comment]
Yes I switched over because I couldn't get this one to work. Sorry!

**Comment by tom95** [comment]
Ah no problem, just wanted to know if it's still urgent. I'll keep the issue open for documentation purposes, but probably only get back to it if it's needed again.

Good luck with your project!
